ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -132,6 +133,9 @@ void py_module(py::module& module) {
 
     auto m_moreh = module.def_submodule("moreh", "moreh operations");
     moreh::bind_moreh_operations(m_moreh);
+
+    auto m_index_fill = module.def_submodule("index_fill", "index_fill operation");
```

Comment:
Please move this to an appropriate location instead of a top level module (like `data_movement`)

Reply:
I think we need to discuss this with TT's code owner first

Reply:
Could you please check this part? The rest looks good.